storybook/src/stories/Forms/VpnServerForm.stories.js

```diff
@@ -0,0 +1,301 @@
+import React from "react";
+import { contains } from "lazy-z";
+import { IcseModal, VpnServerForm } from "icse-react-assets";
+
+export default {
+  component: VpnServerForm,
+  title: "Components/Forms/VpnServerForm",
+  argTypes: {
+    data: {
+      summary: "An optional object describing the tooltip for a title",
+      type: { required: false }, // required prop or not
+      control: "none",
+    },
+    "data.name": {
+      description: "A string value of the name of the VPN server",
+      control: "none",
+      type: { required: false }, // required prop or not
+    },
+    "data.certificate_crn": {
+      description:
+        "The Secrets Manager Certificate CRN of secret from Secrets Manager for this VPN server.",
+      control: "none",
+      type: { required: true }, // required prop or not
+    },
+    "data.method": {
+      description: "The type of authentication (certificate or username).",
+      control: "none",
+      type: { required: true }, // required prop or not
+    },
+    "data.client_ca_crn": {
+      description:
+        "The Client Secrets Manager Certificate CRN to use for the VPN client certificate authority (CA).",
+      control: "none",
+      type: { required: true }, // required prop or not
+    },
+    "data.client_ip_pool": {
+      description:
+        "A function that checks for overlap between the VPN Client CIDR and any existing address prefixes in the VPC or any of the following reserved address ranges: - `127.0.0.0/8` (IPv4 loopback addresses) - `161.26.0.0/16` (IBM services) - `166.8.0.0/14` (Cloud Service Endpoints) - `169.254.0.0/16` (IPv4 link-local addresses) - `224.0.0.0/4` (IPv4 multicast addresses). The prefix length of the client IP address pool's CIDR must be between `/9` (8,388,608 addresses) and `/22` (1024 addresses). A CIDR block that contains twice the number of IP addresses that are required to enable the maximum number of concurrent connections is recommended.",
```

Comment:
i dont think this is a function

Reply:
also should probably be shortened if possible

Reply:
@jvallexm mentioned that the user would pass in a function that would check overlap here so I wasn't sure how to phrase it. Shall I remove the requirements for the overlap and condense it to this instead? 

**_"A function that checks the VPN Client CIDR for overlap between any existing address prefixes in the VPC or any of the reserved address ranges listed in the [IBM Cloud terraform documentation for VPN Servers](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/is_vpn_server#client_ip_pool). A CIDR block that contains twice the number of IP addresses that are required to enable the maximum number of concurrent connections is recommended."_**

Reply:
why would something in `data` be a function? 

this is the schema for vpn_servers, says that client_ip_pool should be a string

```
  "vpn_servers": {
    "Array": {
      "name": "string",
      "certificate_crn": "string",
      "method": "string",
      "client_ca_crn": "string",
      "client_ip_pool": "string",
      "client_dns_server_ips": "string",
      "client_idle_timeout": "number",
      "enable_split_tunneling": "boolean",
      "port": "number",
      "protocol": "string",
      "resource_group": "string",
      "security_groups": "Array<string>",
      "subnets": "Array<string>",
      "vpc": "string",
      "routes": {
        "Array": {
          "name": "string", 
          "action": "string",
          "destination": "string"
        }
      }
    }
  }
```

Reply:
in this case the user is the person using the form

it should be a prop outside of data that is a function that is (optionally) used to check for overlapping ips